### PR TITLE
Fix alphabetize imports

### PR DESCRIPTION
**Pull request by timfrijtersppw**
In this PR I've alphabetized some imports to avoid potential linting errors when generating files with angular-cli.

Also see:
https://palantir.github.io/tslint/rules/ordered-imports/

### CODE REVIEW

**Review by gbumanzordev — APPROVED** [review]

**Comment by timfrijtersppw** [comment]
Updated the PR to master, all ci checks pass. I only don't know how to add the PR target label to this PR. Anyone an idea?

File: packages/schematics/angular/application/files/src/test.ts.template [hunk 1]
```diff
@@ -1,11 +1,11 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
 
-import 'zone.js/dist/zone-testing';
 import { getTestBed } from '@angular/core/testing';
 import {
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting
 } from '@angular/platform-browser-dynamic/testing';
+import 'zone.js/dist/zone-testing';
```
Comment (clydin) [inline packages/schematics/angular/application/files/src/test.ts.template:8]: This is a side effect inducing import that affects global state and should be first.  Can you revert this change?

**Review by clydin — CHANGES_REQUESTED** [review]

**Review by clydin — CHANGES_REQUESTED** [review]
Please also change the commit message scope to `@schematics/angular`.

**Comment by timfrijtersppw** [comment]
Changed the commit message scopes & reverted the test.ts change

**Comment by clydin** [comment]
Looks like there's an accidental `.` in the commit scope.  Otherwise, LGTM.

**Comment by timfrijtersppw** [comment]
> Looks like there's an accidental `.` in the commit scope. Otherwise, LGTM.

Wups, my bad. Renamed the commit message again 😄

**Review by clydin — APPROVED** [review]

**Pull request merged**